"Maximum call stack size exceeded" on circular reference inside object (#1)
mia-platform/json-editor: README.md

    <td>Preserve value at Move Up or Down.(No value is selected automatically upon deletion.)</td>
    <td><code>true</code></td>
  </tr>
  <tr>
    <td>max_depth</td>

>     <td>Max depth of an object that have to be rendered. The missing of this option could cause "maximum call stack size exceeded" in case of recursive object properties</td>

Reviewer: ```suggestion
    <td>Max depth of the nested properties to be rendered of provided json schema. The missing of this option could cause "maximum call stack size exceeded" in case of object properties with circular references</td>
```

>     <td><code>infinite</code></td>

Reviewer: ```suggestion
    <td><code>undefined</code></td>
```

  </tr>
  <tr>
    <td>use_default_values</td>

>     <td>True if have to be used default values based on the "type" of the property</td>

Reviewer: ```suggestion
    <td>If true default values based on the "type" of the property will be used</td>
```
